15.0.Dockerfile

```
@@ -139,6 +139,9 @@ RUN build_deps=" \
         watchdog \
         wdb \
     && (python3 -m compileall -q /usr/local/lib/python3.8/ || true) \
+    # generate flanker cached tables during install when /usr/local/lib/ is still intended to be written to
+    # https://github.com/Tecnativa/doodba/issues/486
+    && python3 -c 'from flanker.addresslib import address' >/dev/null \
```

That last part is trying to silent the warnings? If so, [it's not working](https://github.com/Tecnativa/doodba/runs/5477143537?check_suite_focus=true#step:7:3341).

Try:

```suggestion
    && python3 -c 'from flanker.addresslib import address' 2> /dev/null \
```

---

thanks for checking, will test locally and push a fixed commit

---

using `python3 -c 'from flanker.addresslib import address' >/dev/null 2>&1` to disregard all stdout and stderr from the import statement